pytest_plugins/markers.py

```diff
@@ -30,4 +30,5 @@ def module_markers():
         "host_parameter: Marks host parameter CLI tests",
         "katello_host_tools: Marks host CLI tests where katello host tools is installed on client",
         "host_subscription: Marks host subscription CLI tests",
+        "puppet_enabled: Marks CLI host tests with enabled puppet",
```

Comment:
understand, these all markers are only for  `cli/test_host.py` 
Because the module no longer uses classes and it comes handy to run only the subset and not specify them by name.

Probably, the better naming would be for cli host modules starting with cli_host_puppet_enabled
and for puppet tests, puppet_enabled would be fine :)

Reply:
agree